src/commands/hosting-channel-create.ts

```diff
@@ -69,6 +70,20 @@ export default new Command("hosting:channel:create [channelId]")
         throw e;
       }
 
+      try {
+        await addAuthDomain(projectId, channel.url);
+      } catch (e) {
+        logLabeledWarning(
+          LOG_TAG,
+          marked(
+            `Unable to add channel domain to Firebase Auth. Visit the Firebase Console at ${consoleUrl(
+              projectId,
+              "/overview"
+            )}`
```

Comment:
I'm curious about this call to action - what am I supposed to be doing on the Console? Is there something more specific we can say? Should we drop the call to action?

Reply:
I am not sure.. but the design docs specify to give them a link to the console? I was hoping you would know go/fh-preview-auth-domains